alanEappen/assignments/twopointers/w1d2/lc1.js

```diff
@@ -0,0 +1,70 @@
+// Problem Statement #
+// Given an array of sorted numbers and a target sum, find a pair in the array whose sum is equal to the given target.
+
+// Write a function to return the indices of the two numbers (i.e. the pair) such that they add up to the given target.
+
+//arr is a sorted array 
+// targetSum is given 
+// We need to find two numbers in the array that would equal the targetSum
+//The function returns an array the index of those two numbers.
+/*
+ leftPtr -> first element index in the array
+ rightPtr -> last element index in the array
+ in while loop with a condition of leftPtr < rightPtr
+totalSum is equal to the sum of left and right ptrs
+ then check if totalSum === to targetsum
+ return [leftPtr,rightPtr]
+ else if totalSum > targetSum
+ rightPtr--
+ else if totalSum < targetSum
+ leftPtr++
```

Comment:
Nice psuedo-code! Don't forget to think about edge cases, and then write some test cases to check those edge cases.